src/components/Account/Account.tsx

```diff
@@ -46,6 +46,8 @@ const Account = ({ panelClassName, panelHeaderClassName, canUpdateEmail = true }
   const exportData = useMutation(exportAccountData);
   const [isAlertVisible, setIsAlertVisible] = useState(false);
   const exportDataMessage = exportData.isSuccess ? t('account.export_data_success') : t('account.export_data_error');
+  const accountData = useAccountStore();
+  const shouldAddPassword = accountData.user?.metadata?.register_source !== 'inplayer' && accountData.user?.metadata?.up !== 'true';
```

Comment:
This check is inplayer specific and shouldn't be done here in the component. I would consider setting up a flag in the account store for this matter. And what does this user?.metadata?.up mean?

Reply:
That it's just a flag for the account so we know if a password has been added it is not necessarily comprehensible to users.